pootle/templates/editor/_toolbar.html

```diff
@@ -4,7 +4,9 @@
   <div class="unit-sort">
     <select id="js-filter-sort">
       <option selected value="default">{% trans "Default order" %}</option>
+      {% if display_priority %}
       <option value="priority">{% trans "Decreasing priority" %}</option>
```

Comment:
I still believe this wording is not clear enough, and something like *Important first* is easier to understand and aligns with the existing wording for other sorting criteria. Thoughts @iafan @dwaynebailey?